crates/uv-interpreter/src/python_query.rs

```diff
@@ -195,6 +179,23 @@ fn find_python(
         }
     }
 
+    if cfg!(windows) && !override_path {
+        // Use `py` to find the python installation on the system.
+        match windows::py_list_paths(selector, platform, cache) {
+            Ok(Some(interpreter)) => return Ok(Some(interpreter)),
+            Ok(None) => {
+                // No matching Python version found, continue searching PATH
+            }
+            Err(Error::PyList(error)) => {
+                if error.kind() == std::io::ErrorKind::NotFound {
+                    debug!("`py` is not installed. Falling back to searching Python on the path");
```

Comment:
This message is outdated. We already tried searching the PATh. We should keep the graceful recovery